generators/app/index.js

```diff
@@ -61,6 +73,10 @@ module.exports = class extends Generator {
                         name: 'server',
                         value: 'server'
                     },
+                    {
```

Comment:
I think the whole `choices` array can be derived/constructed from the keys of `this.generators` that you've defined above.

Reply:
done ;)